components/locid/src/subtags/language.rs

```diff
@@ -38,11 +38,12 @@ use tinystr::{tinystr, TinyAsciiStr};
 /// but that form has not been used and ICU4X does not support it right now.
 ///
 /// [`unicode_language_id`]: https://unicode.org/reports/tr35/#unicode_language_id
-#[derive(Default, Debug, PartialEq, Eq, Clone, Hash, PartialOrd, Ord, Copy)]
-pub struct Language(Option<TinyAsciiStr<{ *LANGUAGE_LENGTH.end() }>>);
+#[derive(Debug, PartialEq, Eq, Clone, Hash, PartialOrd, Ord, Copy)]
+pub struct Language(TinyAsciiStr<{ *LANGUAGE_LENGTH.end() }>);
 
 const LANGUAGE_LENGTH: RangeInclusive<usize> = 2..=3;
-const UND_VALUE: TinyAsciiStr<3> = tinystr!(3, "und");
+// Safe because "und" is a valid language subtag
+const UND: Language = Language(unsafe { TinyAsciiStr::from_bytes_unchecked(*b"und") });
```

Comment:
This could be `lang!("und")` after #1631 . @zbraniecki please take a look at that.

Reply:
I'll add a `TODO(#348)` here, and you can fix it in your PR (which it looks like you need to rebase anyway).